packages/soxr/patches/0001-Fix-function-signatures.patch

```diff
@@ -0,0 +1,97 @@
+From ebe422ceaf44c240bfc8e43a7e1dc023d9d584ba Mon Sep 17 00:00:00 2001
+From: swnf <50806201+swnf@users.noreply.github.com>
+Date: Sat, 2 Nov 2024 13:19:32 +0100
+Subject: [PATCH] Fix function signatures
```

Comment:
Could you upstream this patch? If you did, please add a link to your PR.

Reply:
I created the PRs chirlu/soxr#13 (dead upstream) and dofuuz/soxr#3 (python soxr fork). I also left a comment in `meta.yml` and cleaned up the patch a bit.